docs/examples/pipelinerun_details.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: steward.sap.com/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: ok-
+spec:
+  jenkinsFile:
+    repoUrl: https://github.com/sap-production/demo-pipelines
+    revision: master
+    relativePath: success/Jenkinsfile
+  runDetails:
+    jobName: myName
```

Comment:
Better use `myJobName`.

However, for examples/tests I prefer values that repeat the field name and have a number at the end, e.g. `jobName1`.